src/interface/state_descriptor.hpp

```diff
@@ -49,8 +49,11 @@ class StateDescriptor {
   explicit StateDescriptor(std::string const &label) : label_(label) {}
 
   template <typename T>
-  void AddParam(const std::string &key, T value) {
+  void AddParam(const std::string &key, T value, const bool include_with_output = false) {
```

Comment:
If you do end up using the boolean, I would recommend changing it to an enum class which would be more descriptive:

```
enum class IncludeWithOutput {
    True,
    False
};

AddParam<std::string>("Parthenon is Awesome", IncludeWithOutput::True);
```

Reply:
That's a good suggestion. Although it would be a non-issue if we just output all parameters of simple type as discussed above.

Reply:
I'm removing the boolean, otherwise it's a good suggestion.